includes/discovery/discovery-protocols.inc.php

```diff
@@ -204,7 +204,15 @@
         $IndexId = key($lldp['lldpNeighborPortIndexId']);
 
         $local_ifName = $lldp['lldpNeighborPortId'][$IndexId];
+
+        //jetstream port names variations
         $local_port_id = find_port_id('gigabitEthernet ' . $local_ifName, null, $device['device_id']);
+        if (! $local_port_id) {
+            $local_port_id = find_port_id('gigabitEthernet' . $local_ifName, null, $device['device_id']);
```

Comment:
This is the same as the line above right?

Reply:
@murrant 
hi , no, it is not same :)
'gigabitEthernet ' with space
'gigabitEthernet' without space

as i stated on my first comment, TP-LINK have 3 naming scheme sent over SNMP
gigabitEthernet [space] 1/0/x,
gigabitEthernet [NoSpace] 1/0/x
FiberEthernet [NoSpace] 1/0/x,